src/libraries/adaptative-curve/ExpLib.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import {WAD_INT} from "../MathLib.sol";
+
+/// @title ExpLib
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice Library to approximate the exponential function.
+library ExpLib {
+    /// @dev ln(2).
+    int256 internal constant LN_2_INT = 0.693147180559945309 ether;
+
+    /// @dev ln(1e-18).
+    int256 internal constant LN_WEI_INT = -41.446531673892822312 ether;
+
+    /// @dev Above this bound, `wExp` is clipped to avoid overflowing when multiplied with 1 ether.
+    /// @dev This upper bound corresponds to: ln(type(int256).max / 1e36) (scaled by WAD, floored).
+    int256 internal constant WEXP_UPPER_BOUND = 93.859467695000404319 ether;
+
+    /// @dev The value of wExp(`WEXP_UPPER_BOUND`).
+    int256 internal constant WEXP_UPPER_VALUE = 57716089161558943949701069502944508345128.422502756744429568 ether;
```

Comment:
this value was changed because the previous value didn't correspond to the bound from the unbounded wExp

Reply:
except that the code is correct (just checked)